cmd/root_test.go

```diff
@@ -516,10 +516,6 @@ func TestNewCommandWithErrors(t *testing.T) {
 			desc: "when the iam authn login query param is bogus",
 			args: []string{"proj:region:inst?auto-iam-authn=nope"},
 		},
-		{
```

Comment:
Why did you remove this testcase?

Reply:
This was a mistake. I'll restore this test.

Reply:
Actually restoring the test I see why I deleted this: I've changed the behavior to just log a warning -- if a user sets http-port without prometheus or health check, we now tell them that we're ignoring http-port but otherwise keep going.